### ISSUE TITLE
[OpenTelemetry.Contrib.Instrumentation.AWS]: issue for next -beta package release.

### ISSUE THREAD

**Issue by rypdal** [open-telemetry/opentelemetry-dotnet-contrib#1268]
# Issue with OpenTelemetry.Contrib.Instrumentation.AWS

Request: next beta release of the OpenTelemetry.Contrib.Instrumentation.AWS package. 

Recently implemented in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1034  functionality is distributed between 2 nuget packages:

**OpenTelemetry.Instrumentation.AWSLambda**: released -> https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1224
**OpenTelemetry.Contrib.Instrumentation.AWS**: not released

We are not owners of the corresponding project and can't create PR for the release of the package. That's why the issue is created instead.
We would like to ask for release of the next -beta version of the **OpenTelemetry.Contrib.Instrumentation.AWS** package.

**Comment by rypdal** [comment]
FYI: @Oberon00

**Comment by rypdal** [comment]
Hello @srprash  and @atatshaw43. 
We would like to ask for the next beta release of the OpenTelemetry.Contrib.Instrumentation.AWS package. Please, see the issue description.

**Comment by Megasware128** [comment]
I noticed the same for OpenTelemetry.ResourceDetectors.Azure. #1182 has been merged for quite a while now but the feature has not been published to NuGet yet

**Comment by rypdal** [comment]
Hi @srprash , @Kielek , @Oberon00. The title and description of this PR has been adapted according to the target package rename. Can we release the renamed package OpenTelemetry.Instrumentation.AWS ?

**Comment by Kielek** [comment]
First part:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/releases/tag/Extensions.AWS-1.3.0-beta.1
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/releases/tag/ResourceDetectors.AWS-1.3.0-beta.1

**Comment by Kielek** [comment]
@rypdal, technical changes are done. Could you please create PR similar to
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1285?

I suppose that 1.1.0-beta.1 should be fine for Instrumentation.AWS.
Probably 1.2.0-beta.1 for Instrumentation.AWSLambda.

**Comment by rypdal** [comment]
> 1.2.0-beta.1 for Instrumentation.AWSLambda

I think we recently didn't do any changes in Instrumentation.AWSLambda package.  Is the version 1.2.0-beta.1 only because of reference package update ?

**Comment by Kielek** [comment]
> > 1.2.0-beta.1 for Instrumentation.AWSLambda
> 
> I think we recently didn't do any changes in Instrumentation.AWSLambda package. Is the version 1.2.0-beta.1 only because of reference package update ?

Yes, but it is only my perspective, I am fine wit keeping 1.1.0-beta.x.

**Comment by Oberon00** [comment]
I think we should bump the version properly as @Kielek suggested initially, just for the sake of simplicity.

**Comment by Oberon00** [comment]
@Kielek Actually you added a functional change, that could even be considered BREAKING (as the resulting resources will change if attributes were added before). We need to remember to amend the changelog for https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1268

**Comment by Kielek** [comment]
I understand your perspective. Could you please include note in release request PR?

**Comment by Kielek** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/releases/tag/Instrumentation.AWS-1.1.0-beta.1
and https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AWS/1.1.0-beta.1#versions-body-tab

**Comment by birojnayak** [comment]
@Kielek adding for my visibility

**Comment by birojnayak** [comment]
@Kielek this needs our internal documentation to be updated (if we decide internally to stick to this package, till that time let's don't GA). We will also ensure to add few more contact persons going forward.
